src/components/button-toggle/button-toggle.scss

```diff
@@ -29,9 +29,44 @@
   }
 }
 
+.carbon-button-toggle--grouped {
+  .carbon-button-toggle__input:checked ~ .carbon-button-toggle__label{
+    border-color: $blue-sky;
+  }
+
+  + .carbon-button-toggle--grouped {
+    margin-left: -1px;
+  }
+
+  label {
+    border-radius: 0;
+  }
+
+  &:first-of-type label {
+    border-radius: 4px 0 0 4px;
```

Comment:
should use `$carbon-button__border-radius` variable

Reply:
so 'border-radius: $carbon-button__border-radius 0 0 $carbon-button__border-radius'?